internal/pkg/featureflags/feature_flags.go

```diff
@@ -57,16 +57,25 @@ type launchDarklyManager struct {
 	version               *version.Version
 }
 
-func Init(version *version.Version, isTest, isDisabledConfig bool) {
+func Init(cfg *v1.Config) {
+	var platformName string
+	if !cfg.IsTest {
```

Comment:
I don't think we need to add this `if`. If it's a test, we can retrieve the platform name, but we just won't use it.

Reply:
Without this `if` this function will panic when called in `test_helpers.go` and `prerunner_test.go` since we make the config that is passed into `Init`. I'm not sure if it's worth it to populate the test configs of these usages with a fake context map since it's not being used in a test anyways.